user_interface/src/components/BoundingBox.vue

```diff
@@ -57,18 +58,23 @@ export default {
       if (action.type == "home/resetBoundingBoxChanges") {
         this.reset();
       }
+      if (action.type == "home/updateBoundingBoxNumbers") {
+        if (this.selected) {
+          this.text = this.$store.state.home.boxIdToSelectNumberMapping.get(this.boxId);
```

Comment:
State can be retrieved from params
<pre>
this.$store.subscribeAction((action, state) => {
      if (action.type == "home/resetBoundingBoxChanges") {
        this.reset();
      }
      if (action.type == "home/updateBoundingBoxNumbers") {
        if (this.selected) {
          this.text = state.home.boxIdToSelectNumberMapping.get(this.boxId);</pre>

Reply:
Done